src/renderer/components/+namespaces/namespace-select-filter.tsx

```diff
@@ -74,26 +76,42 @@ export class NamespaceSelectFilter extends React.Component<SelectProps> {
   }
 
   onChange([{ value: namespace }]: SelectOption[]) {
-    if (namespace) {
+    if (NamespaceSelectFilter.isMulti && namespace) {
       namespaceStore.toggleContext(namespace);
+    } else if (!NamespaceSelectFilter.isMulti && namespace) {
```

Comment:
I think we should automatically close the namespace select if user selects just one namespace.

Reply:
Agreed.